@navikt/core/react/src/form/combobox/FilteredOptions/FilteredOptions.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { forwardRef } from "react";
+import cl from "clsx";
+import { BodyShort, Label, Loader } from "../../..";
+import { PlusIcon } from "@navikt/aksel-icons";
+import CheckIcon from "./CheckIcon";
+import { useFilteredOptionsContext } from "./filteredOptionsContext";
+import { useSelectedOptionsContext } from "../SelectedOptions/selectedOptionsContext";
+import { useInputContext } from "../Input/inputContext";
+
+interface FilteredOptionsProps {
+  id: string;
+  toggleOption: (e: React.MouseEvent<HTMLLIElement, MouseEvent>) => void;
+  focusInput: () => void;
+  ref: React.RefObject<HTMLUListElement>;
+  singleSelect?: boolean;
+}
+
+const FilteredOptions = forwardRef<HTMLUListElement, FilteredOptionsProps>(
+  ({ toggleOption, singleSelect }, ref) => {
+    const {
+      inputProps: { id },
+      value,
+    } = useInputContext();
+    const {
+      isLoading,
+      isListOpen,
+      filteredOptions,
+      filteredOptionsIndex,
+      filteredOptionsRef,
+      isValueNew,
+    } = useFilteredOptionsContext();
+    const { selectedOptions } = useSelectedOptionsContext();
+
+    return (
+      <ul
+        ref={filteredOptionsRef}
+        className={cl("navds-combobox__list", {
+          "navds-combobox__list--closed": !isListOpen,
+        })}
+        id={`${id}-filtered-options`}
+        role="listbox"
+      >
+        {isLoading && (
+          <li
+            className="navds-combobox__list-item navds-combobox__list-item--loading"
+            role="option"
+            aria-selected={false}
+            id={`${id}-is-loading`}
+          >
+            <Loader />
```

Comment:
Bør Loader ha `aria-hidden` her?

Reply:
Jeg er usikker på hvordan screen-reader experience er best for veldig kortvarige ting? (jeg ville gjettet at ingenting er bedre en kortvarig støy? dvs ja til `aria-hidden` 🤔 ). Ville kanskje spurt Morten hva han tenker? Godt spørsmål!

Reply:
Tror vi har testet det med Morten, men er ikke helt sikker.

I utgangspunktet er jeg litt skeptisk til å skjule den: isLoading er en prop satt på komponenten for tilfeller der det tar tid å få svar på søket. Foreløpig har vi derfor eksplisitt koblet den til input med aria-describedby.

Men kan godt ta en runde på det på mandag. Hvis vi skal skjule den, så gjør vi det bedre med å ikke bruke aria-describedby i dette tilfellet.